features/upgrade/storage/upgrade.feature

```diff
@@ -139,3 +139,32 @@ Feature: Storage upgrade tests
       | ls | /mnt/storage |
     Then the output should contain:
       | test-upgrade |
+
+  # @author wduan@redhat.com
+  # @case_id OCP-31331
+  @upgrade-prepare
+  @users=upuser1,upuser2
+  @admin
+  Scenario: Cluster operator storage should be in correct status after upgrade
```

Comment:
@duanwei33 I think you mean `before upgrade`, right?

Reply:
@pruan-rht  I followed the upgrade case syntax, all added actually one case, a half part is for prepare data and the other is for check after the upgrade, but this is one caseid. But thanks for the reminder, I do miss one keyword(prepare) in line 148.